backend/src/main/java/com/benine/backend/http/CreatePresetHandler.java

```diff
@@ -86,7 +87,7 @@ public void handle(String s, Request request, HttpServletRequest req, HttpServle
    * @throws StreamNotAvailableException  If the camera does not have a stream.
    * @throws IOException  If the image cannot be written.
    */
-  private void createImage(int cameraID, int presetID) throws
+  private static void createImage(int cameraID, int presetID) throws
```

Comment:
I don't think this method belongs here anymore, since the responsibility of this class is handling the http request, not creating preset images. Maybe we should have a static factory for presets so you can just call Factory.getPreset(<parameters>) or something.
What do the other team members think about this?